Jenkinsfile

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'govuk_navigation_helpers'
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  try {
+    stage('Checkout') {
+      checkout scm
+    }
+
+    stage('Clean') {
+      govuk.cleanupGit()
+      govuk.mergeMasterBranch()
+    }
+
+    stage('Bundle') {
+      echo 'Bundling'
+      sh("bundle install --path ${JENKINS_HOME}/bundles/${JOB_NAME}")
+    }
+
+    stage('Linter') {
+      govuk.rubyLinter()
+    }
+
+    stage('Tests') {
+      govuk.contentSchemaDependency('${SCHEMA_GIT_COMMIT:-"deployed-to-production"}')
+      govuk.setEnvar('RAILS_ENV', 'test')
+      govuk.runTests('spec')
+    }
+
+    stage('Publish Gem') {
+      if(env.BRANCH_NAME == "master") {
+        sh("bundle exec rake publish_gem --trace")
```

Comment:
This can be simplified a bit with `govuk.runRakeTask`.

Actually, this is probably worth extracting to the helper script in its entirety, since we'll need to run exactly the same command for other gems.